index.html

```diff
@@ -1251,35 +1271,48 @@ <h3>Semantic Interoperability</h3>
       <section>
         <h2>Terms of Use</h2>
 
-        <p>Terms of use can be utilized by an <a>issuer</a>, a <a>subject</a>, or a
-        <a>holder</a> to constrain the use of information expressed by the Verifiable
-        Credentials Data Model. The <a>issuer</a> places their terms of use inside
-        the <a>credential</a> before it is converted into a <a>verifiable credential</a>.
-        The <a>holder</a> places holder terms of use inside a <a>presentation</a>.
+        <p>
+          Terms of use describe policy describing how a
+          <a>verifiable credential</a> should be used or distributed.
+          For example, terms of use can be added to declare that a credential
+          should not be stored to a database or shared by a holder to another
```

Comment:
`stored to a database` => `stored in a database`
`holder to another` => `holder with another`

Reply:
I disagree with this change as it has deleted important information. For example, it says nothing about adding terms of use to a verifiable presentation, which the original text did.
This is a regressive edit as it removes clarity.

Reply:
@dlongley okay I'll make those changes in a few

@David-Chadwick Good point, I'll add information about terms of use for a verifiable presentation.

Reply:
Okay.  I pushed information about the verifiable presentation and also fixed the grammar nits.  Hope that solves it!